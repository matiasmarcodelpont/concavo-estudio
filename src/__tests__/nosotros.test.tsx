import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

import Nosotros from '@/app/nosotros/page'

describe('Nosotros', () => {
  it('renders nosotros unchanged', () => {
    const { container } = render(<Nosotros />)
    expect(container).toMatchSnapshot()
  })
})
