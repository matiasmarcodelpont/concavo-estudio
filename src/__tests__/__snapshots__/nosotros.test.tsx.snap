// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nosotros renders nosotros unchanged 1`] = `
<div>
  <main>
    <h1>
      Nosotros
    </h1>
  </main>
</div>
`;
